src/bin/sol-fbp-runner/main.c

```diff
@@ -149,12 +181,194 @@ parse_args(int argc, char *argv[])
     return true;
 }
 
+#ifdef USE_MEMMAP
+static void
+clear_memory_maps(void)
+{
+    const struct sol_str_table_ptr *iter;
+    struct sol_memmap_map *map;
+    int i;
+
+    SOL_PTR_VECTOR_FOREACH_IDX (&memory_maps, map, i) {
+        for (iter = map->entries; iter->key; iter++) {
+            free((void *)iter->val);
+        }
+
+        free(map);
+    }
+
+    sol_ptr_vector_clear(&memory_maps);
+}
+
+static bool
+load_memory_map_file(const char *json_file)
```

Comment:
Fair, but I won't be addressing it for v3 - there's a possibility that this json will be merged with conffile - if that happens, splitting will be performed ther. Now that you are here, do you have any opinion on that?